tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md

```diff
@@ -1,138 +1,126 @@
 ---
-title: Take a Tour of SAP BTP Trial
-description: Get familiar with the SAP Business Technology Platform basics to make the most of your trial experience.
+parser: v2
+author_name: Annika Gonnermann
+author_profile: https://github.com/AnnikaGonnermann
 auto_validation: true
 time: 10
 tags: [ tutorial>beginner, topic>cloud, software product>sap-business-technology-platform]
 primary_tag: software product>sap-business-technology-platform
-author_name: Jessica Katz
-author_profile: https://github.com/Jessica-Katz
 ---
 
+# Take a Tour of SAP BTP Trial
+<!-- description --> Get familiar with the SAP Business Technology Platform basics to make the most of your try-out experience.
+
 ## Prerequisites
- - You must have registered for a trial account on SAP Business Technology Platform.
-[Sign up](hcp-create-trial-account) for a free trial.
-
-## Details
-### You will learn
-  - What SAP BTP trial offers
-  - The relationship between global accounts, directories, and subaccounts
-  - What Entitlements and Quotas are
-  - Environments available for BTP trial
-  - How to navigate in the SAP BTP cockpit
-  - What the btp CLI is
+ - You must have registered for a trial account on SAP Business Technology Platform. See: [Get a Free Account on SAP BTP Trial](hcp-create-trial-account) .
+
+## You will learn
+  - What SAP Business Technology Platform (BTP) trial offers
+  - About the relationship between global accounts, directories, and subaccounts
+  - What entitlements and quotas are
+  - About environments available for SAP BTP trial
+  - How to navigate SAP BTP cockpit
+  - What the SAP BTP command line interface (btp CLI) is
 
 ---
 
-[ACCORDION-BEGIN [Step: ](Welcome to SAP BTP Trial)]
+### Welcome to SAP BTP Trial
 
-So you've signed up for SAP BTP trial but are not quite sure where to start? This tutorial is here to help!
+So, you've signed up for SAP BTP trial but are not quite sure where to start? This tutorial is here to help!
 
-A trial account on SAP Business Technology Platform enables you to experience it for free, offering access to a comprehensive set of platform services, as well as early access to beta functionality.
+A trial account on SAP Business Technology Platform enables you to experience the platform for free for a limited amount of time, offering access to a comprehensive set of platform services, as well as early access to beta functionality. Trial accounts are intended for personal exploration, and not for production use or team development. They allow restricted use of the platform resources and services.
 
-Trial accounts are intended for personal exploration, and not for production use or team development. They allow restricted use of the platform resources and services.
+**Note:** You want to explore SAP Business Technology Platform without time constraints? See [Get an Account on SAP BTP to Try Out Free Tier Service Plans](btp-free-tier-account)
```

Comment:
Please remove the label "note" or replace it with a different one. The formatting already makes it a note.

Reply:
Done